mkGraal.nix

```diff
@@ -48,7 +48,7 @@ let
     ++ lib.optionals gtkSupport [ cairo glib gtk3 ];
 
   muslPath = lib.makeBinPath [
-    musl.dev
+    musl.dev gcc
```

Comment:
It must be `stdenv.cc` in this case, to ensure that we are using the same GCC as the current `stdenv` used during build.

Reply:
Huh... Wait, this is macOS. macOS may be using `stdenv.cc` as `Clang` instead.

Reply:
I wasn't testing this on macOS. I was testing on Debian + nix.

Reply:
I can test with stdenv.cc